### PR TITLE
templates: addition of language selector

### PR DESCRIPTION
**Pull request by JavierDelgadoFernandez**
* Adds links to the template (inside the footer section) to set the
  language of the session.

Co-Authored-By: Jiri Kuncar <jiri.kuncar@cern.ch>
Signed-off-by: Javier Delgado <javier.delgado.fernandez@cern.ch>

### CODE REVIEW

File: invenio_theme/templates/invenio_theme/footer.html [hunk 1]
```diff
@@ -25,8 +25,39 @@
...
+        <hr />
+
+        <ul class="list-inline">
+          {% if config['I18N_LANGUAGES'] %}
+            {% from "invenio_i18n/macros/language_selector.html" import language_render %}
+            <li>
+              <div class="dropdown float-right">
+                <button class="btn btn-default dropdown-toggle" type="button"
+                      data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                  {% trans %} Language {% endtrans %}
```
Comment (jirikuncar) [inline invenio_theme/templates/invenio_theme/footer.html]: there should not be space around `Language` inside `{% trans %}`

**Comment by JavierDelgadoFernandez** [comment]
@jirikuncar Rebased and updated.

File: invenio_theme/templates/invenio_theme/footer.html [hunk 2]
```diff
@@ -25,8 +25,21 @@
   <div class="container">
     <div class="row">
       <div class="col-sm-12 col-md-12" style="text-align: right;">
-      <hr />
-      {% trans link="http://inveniosoftware.org" %}Powered by <a href="{{link}}">Invenio</a>{% endtrans %}
+
```
Comment (jirikuncar) [inline invenio_theme/templates/invenio_theme/footer.html]: can you remove all empty lines?

**Comment by JavierDelgadoFernandez** [comment]
@jirikuncar Done.

**Pull request merged**